Jasmine/Common/Constants.swift

```diff
@@ -78,6 +78,14 @@ enum Constants {
 
             static let tileFallInterval: TimeInterval = 0.5
         }
+
+        enum GameOver {
```

Comment:
I think the function is better isn't it?

This will also refactor the `switch case` statement in the VC